### ISSUE TITLE
DateTimeFormatter for HijrahChronology

### ISSUE THREAD

**Issue by Khudoyshukur** [ThreeTen/threetenbp#166]
DateTimeFormatter giving different results;

`val formatter = DateTimeFormatter.ofPattern("MMMM - yyyy").withChronology(HijrahChronology.INSTANCE);`
`val now = HijrahChronology.INSTANCE.date(LocalDate.of(2022, 8, 29));`
`println(formatter.format(now))`

The result is `February 1444`;
Actually it should be `Safar 1444`;

I tested code like above with java.time.DateTimeFormatter and it worked as expected.

**Comment by jodastephen** [comment]
You are right that `HijrahChronology` should work, but it needs a lot of work to do so. Lots of work happened on the JDK's `HijrahChronology` after this project forked, and no-one can just copy the changes.

The text is also much more complete in the JDK than here, which again would be a large thing to fix.
